pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -111,6 +111,7 @@ social:
           %br/
           %div.banner-message
             = hoc_s(:hoc_live_sign_up_message)
+            = hoc_s(:hoc_live_episodes_link, markdown: :inline, locals: {episodes_url: 'https://www.youtube.com/playlist?list=PLzdnOPI1iJNfSMYuR-d7C1wRhluXX7j7v'})
```

Comment:
these should be wrapped in a paragraph; as it stands now, we have plain text being inserted directly into a div, which is not correct

Reply:
done